dbdiagram_syntax/dbdiagram_syntax.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+
+if __name__ == '__main__':
+
+    """
+    This is a complementary program to the psql_ops/create_diagram/create_diagram.py file. That file creates
+    table syntax for sql in the form of txt files. This script will edit those txt files
+    so that they are more akin to the syntax of dbdiagram.io.
+
+    This script will collect all txt files in the directory input, and
+    edit them to approximate the syntax for tables in db diagram.
+
+    """
+
+    txts_location = input("Input the path where txt files are stored: ")
+
+    db_diagram_location = input("Input the path where the new dbdiagram txt files will be stored: ")
```

Comment:
This could be both arguments from a function

Reply:
I edited the script based on your comments! Let me know what you think